layouts/shortcodes/members_list_no_nav.html

```diff
@@ -0,0 +1,47 @@
+
+<!-- 
+  Copyright (c) 2019 Eclipse Foundation, Inc.
+
+  This program and the accompanying materials are made available under the
+  terms of the Eclipse Public License v. 2.0 which is available at
+  http://www.eclipse.org/legal/epl-2.0.
+
+  Contributors:
+    Eric Poirier <eric.poirier@eclipse-foundation.org>
+
+  SPDX-License-Identifier: EPL-2.0
+-->
+<div class="row tabs-sidebar">
+  {{ $col_class := "col-sm-24" }}
+  <div class="{{ $col_class }}">
+    <div class="tab-content background-white no-border">
```

Comment:
The alignment seems off in the preview. I think you can fix that by removing all the classes that we don't need.

Reply:
Done